### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by realatharv**

### CODE REVIEW

File: domains/atharv.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal website for Atharv",
+  "repo": "https://github.com/realatharv/atharv",
+  "owner": {
+    "username": "realatharv",
+    "email": "atharv.sbiz@gmail.com",
+    "twitter": "realatharv"
+  },
+  "record": {
+    "CNAME": "github-username.github.io"
```
Comment (MaskDuck) [inline domains/atharv.json:10]: 404?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/atharv.json (hunk 1 above)
Comment (MaskDuck) [reply domains/atharv.json:10]: you need to replace `github-username` with ur gh username.

Comment (realatharv) [reply domains/atharv.json:10]: oh wait sorry

Comment (realatharv) [reply domains/atharv.json]: i changed it

Comment (MaskDuck) [reply domains/atharv.json]: 404 still exist

**Review by asrvd — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**